.github/workflows/nosimage.yml

```diff
@@ -0,0 +1,41 @@
+name: NOSImage validation script
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
recommended to chose a more random time of day per https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
Done.